fix(gorgone): Add missing centengine report part to centreon-storage database (#1522)
centreon/centreon-collect: gorgone/docs/modules/centreon/statistics.md

## Description

> This module aims to deal with statistics collection of Centreon Engine and Broker. It require action module to be configured on each poller and on the central.

Reviewer: ```suggestion
This module aims to manage the collection of Centreon Engine and Broker statistics. It requires the configuration of an action module on each poller and on the central.

```

> This module aims to manage the collection of Centreon Engine and Broker statistics. It require requires the configuration of an action module on each poller and on the central.

Reviewer: ```suggestion
This module aims to manage the collection of Centreon Engine and Broker statistics. It requires the configuration of an action module on each poller and on the central.

```

## Configuration

| Directive        | Description                                                                                    | Default value                     |
| :--------------- | :--------------------------------------------------------------------------------------------- | :-------------------------------- |
| broker_cache_dir | Path to the Centreon Broker statistics directory (local) use to store node's broker statistics | `/var/lib/centreon/broker-stats/` |

> The configuration needs a cron definition to unsure that statistics collection will be done regulary.

Reviewer: ```suggestion
The configuration needs a cron definition to ensure that statistics are collected regularly.

```

> The configuration needs a cron definition to ensure that statistics collection are collected regularly.

Reviewer: ```suggestion
The configuration needs a cron definition to ensure that statistics are collected regularly.

```

## Events

| Event              | Description                                       |
|:-------------------|:--------------------------------------------------|
| STATISTICSREADY    | Internal event to notify the core                 | 
| STATISTICSLISTENER | Internal Event to receive data from action module |
| BROKERSTATS        | Collect Centreon Broker statistics files on node  |
| ENGINESTATS        | Collect Centreon engine statistics on node        |

## API

### Collect Centreon engine statistics on every nodes configured

> The api send back a token to monitor advancement. Please note this token don't allow to monitor the whole process, only the first part until action command are sent.

Reviewer: ```suggestion
The API returns a token to monitor the advancement. Please note this token does not allow to monitor the whole process but only the first part until an action command is sent.

```
